### PR TITLE
Fix json encode stack strace

### PR DESCRIPTION
**Pull request by fabdsp**
The stack strace can contain Objects and Ressources - json encode fail with invalid type.

### CODE REVIEW

**Comment by cedvan** [comment]
:+1:

**Comment by omansour** [comment]
:+1:

File: EventListener/ExceptionListener.php [hunk 1]
```diff
@@ -99,6 +101,18 @@ public function onKernelException(GetResponseForExceptionEvent $event)
 
         if ($this->stackTrace) {
             $data['error']['stack_trace'] = $exception->getTrace();
+
+            // Clean stacktrace to avoid circular reference ou invalid type
```
Comment (mikaelrandy) [inline EventListener/ExceptionListener.php]: s/ou/or/

Comment (fabdsp) [inline EventListener/ExceptionListener.php]: :lollipop:

**Comment by lnahiro** [comment]
:+1:

**Pull request merged**